source/qb64pe.bas

```diff
@@ -3637,23 +3637,34 @@ DO
     'also check if such a handle was defined and mark it as used then.
     IF INSTR(a3u$, qb64prefix$ + "EMBEDDED$") > 0 THEN
         spo = INSTR(a3u$, qb64prefix$ + "EMBEDDED$"): ket = 0
+        ppo = INSTR(spo + 1, a3u$, "("): IF ppo = 0 THEN ppo = LEN(a3u$) + 1
```

Comment:
Testing locally, this doesn't catch the `Function get_embedded$(a$)` type of case.

Reply:
Right, it needs a check to the left side to and also must go inside the loop.